packages/react-native-editor/__device-tests__/gutenberg-editor-image-@canary.test.js

```diff
@@ -7,15 +7,15 @@ import testData from './helpers/test-data';
 
 describe( 'Gutenberg Editor Image Block tests', () => {
 	it( 'should be able to add an image block', async () => {
-		await editorPage.addNewBlock( blockNames.image );
-		await editorPage.closePicker();
+		// iOS only test - Can only add image from media library on iOS
+		if ( ! isAndroid() ) {
```

Comment:
This test was failing when tested locally on Android then realized that it's actually an iOS-only test, moving the condition to the start of the test so it doesn't have to run anything on Android.